vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -904,7 +901,9 @@ public void setColumnWidth(String columnWidth) {
      * {@link #setAutoResponsive(boolean)} is enabled. The value must be
      * provided with a {@link Unit}, e.g. {@code 100} and {@link Unit#PIXELS}.
      * <p>
-     * By default, the web component uses a width of {@code 13em}.
+     * When the column width is {@code null}, the web component defaults to
+     * {@code 12em} or uses the value of {@code --vaadin-field-default-width`}
```

Comment:
nit: extra backtick.

```suggestion
     * {@code 12em} or uses the value of {@code --vaadin-field-default-width}
```